Metrics/README.md

```diff
@@ -0,0 +1,15 @@
+## Metrics Windows agent building and testing
+
+These scripts are written to be used as [Jenkins](https://jenkins.io/) jobs in an automated CI. Some variables will be taken from the `$env`, passed down from either a [Gearman](http://gearman.org/) or another upstream Jenkins job.
+
+The process of testing the Metrics builds is done with the `start-windows-build.ps1` script and it can be broken into the following parts:
+
+* Install all the prerequisites necessary for the Windows Metrics build: Golang, git, and 7Zip
```

Comment:
I am thinking to remove this install prerequisites phase and commit a script that installs all the required prerequisites. This should be a one time script call at the beginning of the build server setup and not needed afterwards.

What do you think about my idea ?

Reply:
That's good idea. 
I am wondering how Packagepanda is going to impact those start-windows-build.ps1 scripts in this repo. Are they going to be replaced by/ or co-exist in some fashion?

Reply:
I'm not sure exactly. These scripts do testing + build.

I know that pkgpanda does build and setup. I'm not sure about the testing part. We'll see in the future. 

However, I think that pkgpanda will be a good candidate for the setup scripts from: https://github.com/dcos/dcos-windows/

Reply:
Given pkgpanda does not cover any testing at all, all those start-windows-build.ps1 in this repo would probably have to be refactored to accommodate the addition of the pkgpanda.
For example, we might not need Install-Prerequisites for all projects any more for some features (meos)  but it's needed for others (like all Golang related projects for their unittests).
Another potential way to do this is to run those unittests inside container, mimicking the way pkgpanda is doing it.

Reply:
Agree. We'll need to refactor for sure. We'll see in the future.